README.md

```diff
@@ -34,11 +44,18 @@ You can sign up for your own app id at [https://dashboard.branch.io](https://das
 
 #### Unity Scene and Branch Parameters
 
-To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `PATH_PREFIX` and `APP_URI` in the properties.
+To allow Branch to configure itself, you must add a BranchPrefab asset to your first scene. Simply drag into your first scene, and then specify your `APP_KEY`, `APP_PREFIX` and `PATH_PREFIX`, `CUSTOM_DOMAIN`, `APP_LINKS` in the properties.
 
-* `branchKey`: This is your Branch key from the dashboard
-* `androidPathPrefix`: This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
-* `branchUri`: This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `Simulate Fresh Installs`: This is flag that enable or desable debud mode (in debug mode e your app will simulate fresh install each time, just for testing)
+* `Test Mode` : Switch set of parameters, if "Test mode" is enabled then app will use "test" set of parameters, else app will use "live" set of parameters
+* `APP_KEY`: This is your Branch key from the dashboard
+* `APP_URI`: <b>Keeped for supporting old realisation of link scheme.</b> This is the URI scheme you would like to use to open the app. This must be the same value as you entered in [the Branch link settings](https://dashboard.branch.io/#/settings/link) as well. Please do *not* include the `://` characters.
+* `PATH_PREFIX`: <b>Keeped for supporting old realisation of link scheme.</b> This is your Branch android path prefux [Read more](https://github.com/BranchMetrics/Android-Deferred-Deep-Linking-SDK/blob/master/README.md#leverage-android-app-links-for-deep-linking)
+* `CUSTOM_DOMAIN` : <b>Keeped for supporting old realisation of link scheme.</b> This is your custom domain.
+* `APP_LINKS` : This is your app links.
```

Comment:
@aaustin Ok i read a bit through the branch sdk stuff and found this method:

`protected ServerResponse doInBackground(Void... voids) {
            if (thisReq_ instanceof ServerRequestInitSession) {
                ((ServerRequestInitSession) thisReq_).updateLinkClickIdentifier();
            }
            //Update queue wait time
            addExtraInstrumentationData(thisReq_.getRequestPath() + "-" + Defines.Jsonkey.Queue_Wait_Time.getKey(), String.valueOf(thisReq_.getQueueWaitTime()));

            //Google ADs ID  and LAT value are updated using reflection. These method need background thread
            //So updating them for install and open on background thread.
            if (thisReq_.isGAdsParamsRequired()) {
                thisReq_.updateGAdsParams(systemObserver_);
            }

            if (thisReq_.isGetRequest()) {
                return kRemoteInterface_.make_restful_get(thisReq_.getRequestUrl(), thisReq_.getGetParams(), thisReq_.getRequestPath(), timeOut_);
            } else {
                return kRemoteInterface_.make_restful_post(thisReq_.getPostWithInstrumentationValues(instrumentationExtraData_), thisReq_.getRequestUrl(), thisReq_.getRequestPath(), timeOut_);
            }
        }`
starting at line 2362 in the Branch.java file.
It says that the update process of the google ads id should be updated in a background thread especially for "open" and "install" events if im correct?
I gotta admit im not a java pro, but maybe tis may be a point to investigate in to solve this google ads id problem?
Maybe im totally wrong :) Just wanted to help you guys out with some ideas from my side. I hope we get this solved soon^^